docker/development/debian/Dockerfile

```diff
@@ -63,18 +65,40 @@ RUN apt-get update -y \
 
 RUN mkdir -p /tmp/py36 \
  && cd /tmp/py36 \
- && wget --quiet https://www.python.org/ftp/python/3.6.12/Python-3.6.12.tgz \
+ && wget --no-check-certificate --quiet https://www.python.org/ftp/python/3.6.12/Python-3.6.12.tgz \
  && tar -xzf Python-3.6.12.tgz \
  && cd Python-3.6.12 \
- && ./configure --enable-optimizations \
+ && ./configure --enable-optimizations -with-lto  --with-pydebug \
```

Comment:
```suggestion
 && ./configure --enable-optimizations -with-lto --with-pydebug \
```

Reply:
Why aren't these flags needed for newer Python versions?

Reply:
Not sure honestly. Ubuntu dropped support for 3.6 so maybe cause of that?

Reply:
Woops! Fair!

Reply:
Maybe we could drop support for 3.6 in OSTk too?

Reply:
I think we can leave it for now, I ran it without those flags and it worked. Sorry I had to fiddle quite a bit to figure out what was working and not. This is clean for now so i think it's okay but in the long run yeah we can eventually drop support for lower python versions.

Reply:
Agreed!